lib/cn/csched_sp3_work.c

```diff
@@ -462,7 +433,7 @@ sp3_work_leaf_len(
         if (kvsets > runlen_min && cn_ns_vblks(&tn->tn_ns) < kvsets) {
             *mark = list_last_entry(head, typeof(*le), le_link);
             *action = CN_ACTION_COMPACT_KV;
-            *rule = CN_CR_LIDXF;
+            *rule = CN_CR_INDEXF;
```

Comment:
Why do we only use the first letter of the next word in these names? I cannot read the enum and know what it means. Tragic.

Reply:
This one in particular needs work to better align with CN_ACTION_*.  I would like to change cn_comp_rule to just cn_rule, and then CN_CR_*  to CN_RULE_*.

Reply:
That would make sense to me.

Reply:
Done.